ppd/ppd.h

```diff
@@ -878,6 +878,26 @@ extern cf_cmyk_t	*ppdCMYKLoad(ppd_file_t *ppd,
 				    cf_logfunc_t log,
 				    void *ld);
 
+// **** New in libppd 2.0.0: Function to check the correctness
+//      of PPD files, originally cupstestppd ****
+extern cups_array_t *ppdTest(int ignore, int warn, char *rootdir,
+			int help, int verbose, int relaxed, int q_with_v, int v_with_q, int root_present,
+			int files, cups_array_t *file_array);
+
+//Error warning overrides...
+typedef enum overrides_e
```

Comment:
Could you move this type definition to the type definition section (enums) of `ppd/ppd.h`.
Could you also change the names to start with `PPD_TEST_` or `ppd_test_`.